js/gated-content/src/assets/svg/Date Icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="18" height="20" viewBox="0 0 18 20">
```

Comment:
Please remove whitespaces from file names `Date Icon.svg ` let's use `date-icon.svg ` instead. Same for the rest of the files